tests/unit/ui/components/TestBadge.js

```diff
@@ -0,0 +1,31 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+
+import Badge from 'ui/components/Badge';
+import Icon from 'ui/components/Icon';
+
+
+describe(__filename, () => {
+  it('renders a badge', () => {
+    const badge = shallow(<Badge label="super badge" />);
+
+    expect(badge).toHaveClassName('Badge');
+    expect(badge.find(Icon)).toHaveLength(0);
+    expect(badge.children()).toIncludeText('super badge');
```

Comment:
It would be nice to see an assertion making sure that `Icon` gets an `alt` property since that's essential for accessibility.